### ISSUE TITLE
"Loading..." screen occurs after leaving video

### ISSUE THREAD

**Issue by Scottydont12112** [yuliskov/SmartYouTubeTV#600]
The "loading..." screen that had appeared previously is back. It occurs after backing out of a video, or when checking related videos.

**Comment by foreveryoungforty** [comment]
Indeed, I also get this message/screen sometimes when backing out on a video.

**Comment by myanbeiy** [comment]
I am experiencing this as well, along with missing microphone in search. Issue started a couple of days ago. Installing the latest version did not help. Tried the beta as well and no change.

**Comment by myanbeiy** [comment]
I am copy/pasting a response I shared on another thread because I found a odd solution to this issue.

For context I am having this issue as well as a couple of others. All symptoms started together a few days ago. Voice search missing. Black "loading" screen on exit video. Save not working. Other weird UI quirks.

You will find this odd, but the problems go away when I VPN and select another location. I am based in California and no matter what I try the app will continue to have the usability problems you mentioned unless I select a non-California location (Chicago, New York, Germany, etc. all work). To double confirm, I selected California VPN servers like Los Angeles and San Francisco and the app immediately loses microphone search again, and shows a "loading" screen when you hit the back button, the exit prompt, all the weird problems...

Go figure.
